client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -29,16 +40,34 @@ class HostCredentialsPage extends Component {
 		this.setFieldValue( fieldName, e.target.value );
 	}
 
+	redirectToCart = () => {
+		upgradesActions.addItem( cartItems.guidedTransferItem() );
+		page.redirect( `/checkout/${ this.props.siteSlug }` );
```

Comment:
Ah interesting, I didn't know about that function. I've left this as is for now, I find it a little easier to read for simple cases like this.